src/server/pages.js

```diff
@@ -2,7 +2,7 @@ import nextRoutes from 'next-routes';
 
 const pages = nextRoutes();
 
-const createOrderPage = process.env.USE_OLD_CREATE_ORDER === 'true' ? 'createOrder' : 'createOrderNewFlow';
+const createOrderPage = process.env.USE_NEW_CREATE_ORDER === 'true' ? 'createOrderNewFlow' : 'createOrder';
```

Comment:
Was something done to ensure the availability of this environment variable (client side)?

Reply:
I didn't ; just added `USE_NEW_CREATE_ORDER: process.env.USE_NEW_CREATE_ORDER || false,` to `_document.js`